src/vectorsynchronizer.js

```diff
@@ -81,6 +81,22 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
     csPrimitives.show = olLayer.getVisible();
   });
 
+  var onOpacityChange = function() {
+    var bbs = csPrimitives.context.billboards;
+    var opacity = olLayer.getOpacity();
+    if (!goog.isDef(opacity)) {
+      opacity = 1;
+    }
+    bbs.olLayerOpacity = opacity;
+    var i, bb;
+    for (i = bbs.length - 1; i >= 0; --i) {
+      bb = bbs.get(i);
+      bb.color = new Cesium.Color(1.0, 1.0, 1.0, bb.olStyleOpacity * opacity);
```

Comment:
The color of the billboard is not used for anything but setting the opacity. See https://github.com/openlayers/ol3-cesium/blob/3e9393c1817d28d8d6b17a437166833d49d0c1ac/src/core.js#L793. If that is a bad way to handle opacity, it should be changed there as well. Maybe something more suited for a follow-up pull request.